src/Components/TimelinePost/TimelinePost.js

```diff
@@ -2,16 +2,22 @@ import RecommendeeCard from "../RecommendeeCard/RecommendeeCard"
 import RecommendeeIcon from "../RecommendeeIcon/RecommendeeIcon"
 import './_TimelinePost.scss'
 
-const TimelinePost = () => {
+const TimelinePost = ({ recommenderAvatar, recommenderName, poster, title, releaseYear, rating, genres, date }) => {
```

Comment:
This is more of a question about global context and using the context API. Since we have a Context API, does that mean we do not have to pass props and just use the useContext instead?

Reply:
We discussed this question as a group, and we can still pass props as needed. Passing props will be a judgment call for the developer on which method is more efficient/ ETC.